File: Assignment__18/Question4.java
import java.util.Scanner;

class function1
{
    int DisplayEvenSum(int iNo)
    {
        int iCnt = 0;
        int iEvenSum = 0;

        if(iNo < 0)
        {
            iNo = -iNo;
        }

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            if(iCnt % 2 == 0)
            {
                iEvenSum = iEvenSum + iCnt;
            }
            
        }
        return iEvenSum;
    }
}

class function2
{
    int DisplayOddSum(int iNo)
    {
        int iCnt = 0;
        int iOddSum = 0;

        if(iNo < 0)
        {
            iNo = -iNo;
        }

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            if(iCnt % 2 != 0)
            {
                iOddSum = iOddSum + iCnt;
            } 
        }
        return iOddSum;
    }
}

class Question4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        int iResult1 = 0;
        int iResult2 = 0;

        function1 fobj1 = new function1();
        
        iResult1 = fobj1.DisplayEvenSum(iValue);

        function2 fobj2 = new function2();
        
        iResult2 = fobj2.DisplayOddSum(iValue);

        System.out.println("The Addition of Even number is : " + iResult1);

        System.out.println("The Addition of odd number is : " + iResult2);
        

    }
}

File: Assignment__13/Question4.c
#include<stdio.h>

int NumAddition(int iNo)
{
    int iCnt = 0;
    int iAdd = 0;
    for(iCnt = 1; iCnt <= iNo; iCnt++)
    {
        iAdd = iAdd + iCnt;
    }

    return iAdd;
}

int main()
{
    int iValue = 0;
    int iResult = 0;

    printf("Enter the number : \n");
    scanf("%d", &iValue);

    iResult = NumAddition(iValue);

    printf("The Addition is : %d\n", iResult);

    return 0;
}